teacher.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'person'
+
+# teacher.rb
+class Teacher < Person
+  def initialize(_id, age, specialization, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission: parent_permission, name: name)
```

Comment:
- **[OPTIONAL]** I also think you should take note of the arrangement of the arguments for the super method. It should match the one in the Person class to avoid mixing up inputs. Name should come before age and parent_permission.

  ```
  super(name: name, age: age, parent_permission: parent_permission)
  ```

Reply:
Fixed thanks